.gitignore

```diff
@@ -4,4 +4,6 @@ node_modules/
 .idea/
 # Sed backup files.
 *-e
+# Vim swap files.
+*.swp
```

Comment:
You may find this useful :smile: : http://stackoverflow.com/questions/7335420/global-git-ignore